packages/web/app/package.json

```diff
@@ -6,8 +6,6 @@
     "build": "tsx ../../../scripts/runify.ts src/server/index.ts && vite build --outDir dist/client",
     "build-storybook": "storybook build",
     "dev": "tsx watch --clear-screen=false --exclude \"./**/*.mjs\" src/server/dev.ts",
-    "generate-changelog": "node ../../../scripts/generate-changelog.js",
-    "postinstall": "pnpm generate-changelog",
```

Comment:
Why?

Reply:
I simplified it to generate changelogs once `feed.xml` will be generated https://github.com/graphql-hive/console/blob/nextra-4--move-files/packages/web/docs/src/app/feed.xml/route.ts